scala-generator/src/main/scala/models/Play2Models.scala

```diff
@@ -46,6 +46,55 @@ trait Play2Models extends CodeGenerator {
       }
     }
 
+    val timeSerde = ssd.config.timeLib match {
```

Comment:
Play 2.4, and up, ships with some JSON readers, and writers for joda, and java time.
It might be a good idea to leverage that code instead of adding more to maintain.

- https://www.playframework.com/documentation/2.4.x/api/scala/index.html#play.api.libs.json.Reads$
- https://www.playframework.com/documentation/2.5.x/api/scala/index.html#play.api.libs.json.Reads$
- https://www.playframework.com/documentation/2.6.x/api/scala/index.html#play.api.libs.json.Reads$

Reply:
Ack. Will do that separately as it'll cause a lot of noise.